### PR TITLE
[RadiusCheckAdmin] New Entries now have new_value field visible

### PR DESCRIPTION
**Pull request by peppelinux**
fixes this:
https://github.com/openwisp/django-freeradius/issues/92
my fault in the last revision, nothing more to say :)

### CODE REVIEW

File: django_freeradius/static/django-freeradius/js/radcheck.js [hunk 1]
```diff
@@ -5,13 +5,21 @@ django.jQuery(function ($) {
...
         newValueInput = newValueRow.find('input'),
         changeId = 'change-radcheck-value',
         changeHtml = '<button class="button" id="' + changeId + '">' + gettext('change') + '</button>',
         cancelId = 'cancel-change-radcheck-value',
-        cancelHtml = '<button class="button" id="' + cancelId + '">' + gettext('cancel') + '</button>';
+        cancelHtml = '<button class="button" id="' + cancelId + '">' + gettext('cancel') + '</button>',
+        rawValue = $(rawValueInput).find('.readonly');
     newValueRow.hide();
-    rawValueRow.find('.readonly').append(changeHtml);
+    console.log(rawValue);
```
Comment (nemesisdesign) [inline django_freeradius/static/django-freeradius/js/radcheck.js]: sems you forgot this

**Comment by peppelinux** [comment]
console.log removed, I should have been born with 4 eyes OR I should not push my changes before lunch time!

Now everything should be ok, JSlint suggested me someother fetish things as well.
It was right :)

**Comment by peppelinux** [comment]
these two commits are bugfixes, not additional features

**Comment by nemesisdesign** [comment]
@peppelinux I'm sorry but I can't accept this sort of complexity just to make a `notes` field prettier.
Let's keep it simple please. The project has other priorities right now.

**Comment by peppelinux** [comment]
Hi nemesis, It's just a overlapping commit, would you merge just the bugfix and exclude textarea resize?
The important commit is relative to bugfix, it would be important to merge if you agree.

**Comment by nemesisdesign** [comment]
Sure! Please send it again with just the fix.

**Comment by peppelinux** [comment]
I am so sorry about this rejected contribution, I'm sure that it's has not been well understood in terms of functionality and style, I would like to tell you that the javascript code related to the textarea field has been thought of as an independent module, it can be applied to any textarea allowing this to make the autoresize based on what the user writes or what was previously written. I have learned that the project has taken a direction that is not entirely clear and transparent to myself and I admit that I feel a bit in the way, bytheway I can openly say that what has been shared so far has been beautiful, useful and fun. Always thanks then, I will follow django-freeradius more from the distance continuing for my needs to develop on my forks. Thank you for everything you showed me and see you soon.

**Pull request closed without merging**